kuma/search/managers.py

```diff
@@ -12,6 +12,10 @@ class FilterManager(models.Manager):
     def visible_only(self):
         return self.filter(visible=True)
 
+    @lru_cache(maxsize=None)
```

Comment:
If using the LRU cache is wanted without maximum size I'd recommend using the `cached_property` decorator instead since it reduces the amount of logic to process per call